_apps/vaquita.md

```diff
@@ -0,0 +1,30 @@
+---
+title: Vaquita, fast and Accurate Identification of Structural Variation Using Combined Evidence
+layout: app
+contact: Jongkyu Kim
+categories: official
```

Comment:
```
header:
  app_image: /assets/images/apps/vaquita_logo.png
```

And move the vaquita_logo.png to the folder

Reply:
ah thx